sql/server.go

```diff
@@ -139,7 +198,10 @@ func (s *Server) exec(req driver.Request) (driver.Response, error) {
 		return resp, err
 	}
 	for _, stmt := range stmts {
-		// TODO(pmattis): Fill placeholders.
+		// Bind all the placeholder variables in the stmt to actual values.
+		if err = s.bindValues(stmt, parameters{params: req.Params}); err != nil {
```

Comment:
I've added an incomplete implementation of WalkStmt. It should be okay to check this in and follow up with another PR that implements it for all statements.